src/AboutScreen/about.jsx

```diff
@@ -5,84 +5,92 @@ import { useSelector } from "react-redux";
 import { getVersion, getBuildNumber } from "react-native-device-info";
 import colors from "../common/colors";
 import STRINGS from "../common/localization";
-import styles from "./styles";
+import { styles, nightStyles } from "./styles";
 import constant from "../common/constant";
 import useHeader from "./hooks/useHeader";
 
 function AboutScreen({ navigation }) {
   const { isNightMode, isStatusBar } = useSelector((state) => state);
   useHeader(navigation);
+  const { backgroundColor } = nightStyles(isNightMode);
+  const { TOOLBAR_COLOR_ALT2 } = colors;
+  const { nightMode, SGTitle, margin10, margin, singleLine, leftContainer, underlayColor } = styles;
+  const {
+    CREATED_BY,
+    SUNDAR_GUTKA,
+    ABOUT_1,
+    ABOUT_2,
+    ABOUT_3,
+    ABOUT_4,
+    ABOUT_5,
+    ABOUT_6,
+    baniDB,
```

Comment:
This `baniDB` is the only string here in camel case, can we change it to uppercase to match others.

Reply:
fixed